kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/modem/SupportedUsbModemInfo.java

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *  Eurotech
  *  3 PORT d.o.o.
- *  Sterwen Technologies
+ *  Sterwen Technology
```

Comment:
The year in copyright header is outdated.

Reply:
OK. I will update it.